Tasks/Common/docker-common/dockercommandutils.ts

```diff
@@ -178,8 +182,11 @@ async function getHistory(connection: ContainerConnection, image: string): Promi
         });
     }
     catch (e) {
-        defer.reject(e);
-        console.log(e);
+        // Swallow any exceptions encountered in executing command
+        // such as --format flag not supported in old docker cli versions
+        output = null;
+        defer.resolve();
+        tl.warning(e);
```

Comment:
Prefix with some message indicating that we are not able to extract layer information.